pyproject.toml

```diff
@@ -44,9 +32,27 @@ classifiers = [
     "Operating System :: OS Independent",
     "Topic :: Text Processing :: Linguistic",
 ]
+include = ["udtube*"]
+exclude = ["configs*",  "examples*", "scripts*" ]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+jsonargparse = { version = "^4.32.0", extras = ["signatures"] }
+lightning = "^2.4.0"
+transformers = "^4.44.0"
+torch = "^2.4.0"
+torchmetrics = "^1.4.0.post0"
+black = "^24.4.2"
```

Comment:
So a distinction is being lost here: this is not a dependency of the library at all and indeed users don't need it. But developers might. That's why it's in `requirements.txt` but not in `pyproject.toml` in the current design. Is there some way to encode that?

Reply:
fixed